src/main/resources/schema.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE products (
+                          id BIGINT AUTO_INCREMENT PRIMARY KEY,
+                          name VARCHAR(255) NOT NULL,
+                          price DOUBLE NOT NULL,
+                          image_url VARCHAR(512) NOT NULL
+);
```

Comment:
name & image_url has limits!

I think adding validation logic to the `Product`, might make the code more robust and easier to test.
U think so too, right? 😄